doc/jcli/transaction.md

```diff
@@ -32,7 +32,12 @@ content information of a transaction:
 
 # Examples
 
-Let's say we have the following utxo
+The following example focuses on using an utxo as input, the few differences when transfering from an account will be pointed out when necessary.
+There is also a script [here](https://github.com/input-output-hk/jormungandr/blob/master/scripts/send-transaction) to send a transaction from a faucet account to a specific address which could be used as a reference.
```

Comment:
I think this script is not really good. The `scripts` directory is more like a playground now:

* `bootstrap` generate everything for someone to start a new blockchain (genesis or bft);
* `bootstrap` produces scripts to test some actions (sending money, sending a certificate...);

the generated scripts can then be used to play with a blockchain and see some properties.

If @vincenthz is ok, I would rename the script directory as _playground_ and write a README in that directory explaining how to use the playground.